packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -8244,4 +8244,45 @@ describeWithDOM('mount', () => {
       expect(spy).to.have.property('callCount', 1);
     });
   });
+
+  describe('hooks', () => {
+    it('will run', () => {
+      const Foo = () => {
+        const [fooValue, setFooValue] = React.useState("hi");
+        const handleChange = () => {
+          setFooValue("hey");
+        }
+        return (
+          <div>
+            <button onClick={() => handleChange}>
```

Comment:
i think you want `() => handleChange()` here? otherwise it won't actually do anything. Also, never use a button without a type :-)
```suggestion
            <button type="button" onClick={handleChange}>
```